test/lib/jdk/test/lib/process/ProcessTools.java

```diff
@@ -222,13 +222,17 @@ protected void processLine(String line) {
 
                 if (latch.getCount() != 0) {
                     if (!p.isAlive()) {
-                        throw new RuntimeException("Started process " + name + " is not alive.");
+                        // Give some extra time for the StreamPumper to run after the process completed
+                        Thread.sleep(1000);
+                        if (latch.getCount() > 0) {
+                            throw new RuntimeException("Started process " + name + " is not alive.");
```

Comment:
Nit: the message is not very informative - we expect the process to die, the problem is it died before giving the necessary output. Suggestion:
> "Started process " + name + "terminated before producing the expected output"